<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>734</width>
    <height>785</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Study with Bunny - Home</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #E1F5FE; /* M√†u n·ªÅn xanh pastel */</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPushButton" name="btn_comeback">
      <property name="text">
       <string>‚Üê Quay l·∫°i</string>
      </property>
      <property name="minimumSize">
       <size>
        <width>100</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFF9C4; border-radius: 15px; font-size: 18px;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <family>Chalkboard SE</family>
        <pointsize>44</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFF9C4;
     border-radius: 30px;
     padding: 10px;
     color: #000000; /* M√†u ch·ªØ h·ªìng ƒë√†o */
     text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.2); /* Hi·ªáu ·ª©ng b√≥ng ch·ªØ */</string>
      </property>
      <property name="text">
       <string>B·∫Øt ƒë·∫ßu b√†i h·ªçc n√†o! üê∞</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="minimumSize">
       <size>
        <width>710</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Chalkboard SE</family>
        <pointsize>36</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FFDCDC;
     border-radius: 35px;
     padding: 10px;
     color: #000000; /* M√†u ch·ªØ h·ªìng ƒë√†o */
     text-shadow: 2px 2px 4px rgba(0, 0, 0, 0.2); /* Hi·ªáu ·ª©ng b√≥ng ch·ªØ */</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btn_start">
      <property name="text">
       <string>B·∫Øt ƒë·∫ßu</string>
      </property>
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #FF69B4; border-radius: 15px; font-size: 18px; color: white;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="progressLabel">
      <property name="font">
       <font>
        <family>Chalkboard SE</family>
        <pointsize>27</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #000000; 
     padding: 5px;</string>
      </property>
      <property name="text">
       <string>H√¥m nay b·∫°n ƒë√£ ho√†n th√†nh 0/5 b√†i h·ªçc! C·ªë l√™n nh√© üê∞</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>734</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>